src/scss/components/_url-box.scss

```diff
@@ -12,10 +12,27 @@
   padding: 0 15px;
   transition: all .3s ease;
   border-radius: 4px;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
   &:hover {
     border-color: $light-grey;
   }
   &:focus, &:active {
     border-color: $primary-color;
   }
 }
+
+.btn.btn-outline-primary.url-box__button {
```

Comment:
I'm guessing this level of specificity is because of clarity UI's highly specific CSS styles?

Reply:
Yes, I'm also not very good with CSS, so this the only way to make the style work